modules/flowable-engine/src/main/java/org/flowable/engine/impl/TaskServiceImpl.java

```diff
@@ -23,50 +23,10 @@
 
 import org.flowable.common.engine.api.FlowableIllegalArgumentException;
 import org.flowable.common.engine.impl.service.CommonEngineServiceImpl;
+import org.flowable.engine.impl.util.ScopedVariableContainerHelper;
 import org.flowable.engine.TaskService;
 import org.flowable.engine.impl.cfg.ProcessEngineConfigurationImpl;
-import org.flowable.engine.impl.cmd.AddCommentCmd;
-import org.flowable.engine.impl.cmd.AddIdentityLinkCmd;
-import org.flowable.engine.impl.cmd.ClaimTaskCmd;
-import org.flowable.engine.impl.cmd.CompleteTaskCmd;
-import org.flowable.engine.impl.cmd.CompleteTaskWithFormCmd;
-import org.flowable.engine.impl.cmd.CreateAttachmentCmd;
-import org.flowable.engine.impl.cmd.DelegateTaskCmd;
-import org.flowable.engine.impl.cmd.DeleteAttachmentCmd;
-import org.flowable.engine.impl.cmd.DeleteCommentCmd;
-import org.flowable.engine.impl.cmd.DeleteIdentityLinkCmd;
-import org.flowable.engine.impl.cmd.DeleteTaskCmd;
-import org.flowable.engine.impl.cmd.GetAttachmentCmd;
-import org.flowable.engine.impl.cmd.GetAttachmentContentCmd;
-import org.flowable.engine.impl.cmd.GetCommentCmd;
-import org.flowable.engine.impl.cmd.GetIdentityLinksForTaskCmd;
-import org.flowable.engine.impl.cmd.GetProcessInstanceAttachmentsCmd;
-import org.flowable.engine.impl.cmd.GetProcessInstanceCommentsCmd;
-import org.flowable.engine.impl.cmd.GetSubTasksCmd;
-import org.flowable.engine.impl.cmd.GetTaskAttachmentsCmd;
-import org.flowable.engine.impl.cmd.GetTaskCommentsByTypeCmd;
-import org.flowable.engine.impl.cmd.GetTaskCommentsCmd;
-import org.flowable.engine.impl.cmd.GetTaskDataObjectCmd;
-import org.flowable.engine.impl.cmd.GetTaskDataObjectsCmd;
-import org.flowable.engine.impl.cmd.GetTaskEventCmd;
-import org.flowable.engine.impl.cmd.GetTaskEventsCmd;
-import org.flowable.engine.impl.cmd.GetTaskFormModelCmd;
-import org.flowable.engine.impl.cmd.GetTaskVariableCmd;
-import org.flowable.engine.impl.cmd.GetTaskVariableInstanceCmd;
-import org.flowable.engine.impl.cmd.GetTaskVariableInstancesCmd;
-import org.flowable.engine.impl.cmd.GetTaskVariablesCmd;
-import org.flowable.engine.impl.cmd.GetTasksLocalVariablesCmd;
-import org.flowable.engine.impl.cmd.GetTypeCommentsCmd;
-import org.flowable.engine.impl.cmd.HasTaskVariableCmd;
-import org.flowable.engine.impl.cmd.NewTaskCmd;
-import org.flowable.engine.impl.cmd.RemoveTaskVariablesCmd;
-import org.flowable.engine.impl.cmd.ResolveTaskCmd;
-import org.flowable.engine.impl.cmd.SaveAttachmentCmd;
-import org.flowable.engine.impl.cmd.SaveCommentCmd;
-import org.flowable.engine.impl.cmd.SaveTaskCmd;
-import org.flowable.engine.impl.cmd.SetTaskDueDateCmd;
-import org.flowable.engine.impl.cmd.SetTaskPriorityCmd;
-import org.flowable.engine.impl.cmd.SetTaskVariablesCmd;
+import org.flowable.engine.impl.cmd.*;
```

Comment:
Project standards are not to use package imports but rather enumerate all the classes.

Reply:
Ok, there now all imports.